ipaserver/install/ipa_cacert_manage.py

```diff
@@ -128,6 +128,7 @@ def run(self):
                 return self.install()
         finally:
             api.Backend.ldap2.disconnect()
+        return None
```

Comment:
``validate_option`` ensures, that this line is not reachable. Please either add a comment or change the fix to something like:

```
             if command == 'renew':
                 return self.renew()
             elif command == 'install':
                 return self.install()
             else:
                 # Unreachable
                 raise NotImplementedError
```